### PR TITLE
[2/4] Add split export to V2 tools drawer

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

Add an "Export data" section to the tools drawer.

Only for V2 projects atm. V1 project coming in the next PR.

## Screenshots or screen recordings

Screen Shot 2022-08-25 at 11 10 56 AM

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/Project/ProjectToolsDrawer/ExportSection/ExportSection.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import { ThemeContext } from 'contexts/themeContext'
+import { useContext } from 'react'
+
+export function ExportSection({
+  exportPayoutsButton,
+  exportReservedTokensButton,
+}: {
+  exportPayoutsButton: JSX.Element | undefined
+  exportReservedTokensButton: JSX.Element | undefined
```
Comment (tomquirk) [inline src/components/Project/ProjectToolsDrawer/ExportSection/ExportSection.tsx:12]: V1 will pass in its own buttons here.

File: src/components/v2/V2Project/V2ProjectToolsDrawer/ExportSplitsButton.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import { DownloadOutlined } from '@ant-design/icons'
```
Comment (tomquirk) [inline src/components/v2/V2Project/V2ProjectToolsDrawer/ExportSplitsButton.tsx:1]: We'll have v1 equivalents of this component.

File: src/components/v2/V2Project/V2ProjectToolsDrawer/ExportSplitsButton.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  'lockedUntil',
+  'projectId',
+  'allocator',
+]
+
+const prepareSplitsCsv = (splits: Split[]): (string | undefined)[][] => {
+  const csvContent = splits.map(split => {
+    return [
+      split.beneficiary,
+      `${parseFloat(formatSplitPercent(BigNumber.from(split.percent))) / 100}`,
```
Comment (tomquirk) [inline src/components/v2/V2Project/V2ProjectToolsDrawer/ExportSplitsButton.tsx:26]: this means values are represented in fractions. i.e `0.0304` = 3.04%

**Comment by johnnyd-eth** [comment]
Same comment as here: https://github.com/jbx-protocol/juice-interface/pull/1823#issuecomment-1226774331

**Review by johnnyd-eth — APPROVED** [review]
L F G

**Pull request merged**